### PR TITLE
Refactor and style

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Hey folks, did some minor changes and refactor. Please review my PR

### CODE REVIEW

File: src/context/AuthProvider.jsx [hunk 1]
```diff
@@ -97,6 +91,15 @@ export default function AuthProvider({ children }) {
...
     }, []);
+    
+    useEffect(() => {
+        if (auth.isAuthenticated) {
+            localStorage.setItem("chess-tv-user-auth", JSON.stringify(auth));
+        }else{
+            localStorage.setItem("chess-tv-user-auth", JSON.stringify(auth));
+
+        }
+    }, [auth]);
```
Comment (archit7448) [inline src/context/AuthProvider.jsx]: what is the difference between if-else   it  is doing the same function

File: src/pages/explorepage/ExplorePage.jsx [hunk 2]
```diff
@@ -30,10 +34,17 @@ export default function ExplorePage() {
...
                 </button>
             </div>
+            {searchValue  && <div className="font-normal">Showing results for {<b className="font-large">{searchValue}</b>}</div>}
             <div className="grand-video-list">
-                {videoFilter(videoState).map((video)=>{
+                {videoList.map((video)=>{
                     return <VideoCard video={video} setVideo={setSelectedVideo}/>
                 })}
+                {videoList.length === 0 && <div className="grand-empty">
+                        {/* <img className="grand-empty-img" src="/images/search_result.png" alt="" /> */}
```
Comment (archit7448) [inline src/pages/explorepage/ExplorePage.jsx]: commented code remove

**Review by archit7448 — APPROVED** [review]
- suggested some changes  
- code looks really good proper naming , structure 
- UI looks good 
- Good to merge

**Review by srishtigautam21 — APPROVED** [review]
the code looks good to merge

**Review by dev-prabhat — APPROVED** [review]

**Pull request merged**